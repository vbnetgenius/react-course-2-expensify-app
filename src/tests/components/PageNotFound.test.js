import React from 'react';
import { shallow } from 'enzyme';
import PageNotFound from './../../components/PageNotFound';
import toJSON from 'enzyme-to-json';

test("should render PageNotFound correctly", () => {
    const wrapper = shallow(<PageNotFound />);
    expect(toJSON(wrapper)).toMatchSnapshot();
});
