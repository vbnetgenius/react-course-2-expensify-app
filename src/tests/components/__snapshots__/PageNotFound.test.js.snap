// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render PageNotFound correctly 1`] = `
<div>
  404: Not found
</div>
`;
